### DISCUSSION TITLE
Help updating a plugin to be compatible with Android 12 please

### DISCUSSION THREAD

**Discussion by PediaCalc1** [apache/cordova#450] [Q&A]
I have a app that depends on this plugin 'https://www.npmjs.com/package/cordova-plugin-zebraprinter' to print to a Zebra blue tooth printer.  It is not compatible with Android 12.  Wondering if anyone is able to update it?

**Comment by breautek** [comment]
Kind of depends on the issue but the plugin includes a closed source library: https://github.com/jesusbaqui/zebraprinter/blob/master/src/android/ZSDK_ANDROID_API.jar

So if the problem is in there, then there isn't much that can be done, unless if you know where that library is sourced.

**Reply by PediaCalc1** [reply]
Disclosure:  I'm a self taught developer and half the time use trial and error and stumble upon a solution.  So I am not sure what is happening in the code, but as the end user, I just get a pop up 'check that printer is on' - since updating to Android 12.  I can go back to Android 10 in my config.xml and it works again.  
Is any of that helpful?
